src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveEx.cs

```diff
@@ -17,6 +17,10 @@ public class ZipArchiveEx : IDisposable
 			get { return zip; }
 		}
 
+		public bool AutoFlush { get; set; } = true;
```

Comment:
Related suggestion from our 1:1: instead of `AutoFlush` causing `ZipArchiveEx` to "flush" contents after a set number of `Add()`s -- 50, iirc -- perhaps it should instead "flush" after a threshold *size* of added files.  The primary point to flushing was to reduce memory consumption, after all…